src/modality-specific-files/genetic-descriptor.md

```diff
@@ -19,11 +19,12 @@ and can be used for practical guidance when curating a new dataset.
 
 ## Dataset Description
 
-Genetic descriptors are encoded as an additional, OPTIONAL entry in the
+If genetic data is supplied as part of a BIDS dataset,
```

Comment:
If genetic information is suppied
--> data are in a biobank, the descriptor tells what info in available

Reply:
```suggestion
If information on associated genetic data is supplied as part of a BIDS dataset,
```